postgres/osmap.yaml

```diff
@@ -0,0 +1,7 @@
+{% import_yaml "postgres/repo.yaml" as repo %}
+
+Fedora:
+  pkg_repo:
+    baseurl: 'https://download.postgresql.org/pub/repos/yum/{{ repo.version }}/fedora/fedora-$releasever-$basearch'
+
+# vim: ft=yaml
```

Comment:
Side comment. If you use (or would like to use) [salt-vim](https://github.com/saltstack/salt-vim) plugin, it's better to set file type to sls here: `vim: ft=sls`. It will enable correct Jijna syntax highlighting.